### PR TITLE
[main] switch to using libcxx-devel

### PR DESCRIPTION
**Pull request by h-vetinari**
For #310, using builds from https://github.com/conda-forge/libcxx-feedstock/pull/183

### CODE REVIEW

**Comment by h-vetinari** [comment]
@isuruf, could you please check this (https://github.com/conda-forge/clangdev-feedstock/pull/311/commits/57b3a4a5e869e278a8a8905b3305460c308fe3de; the rest will be removed again) is what you had in mind?

**Comment by h-vetinari** [comment]
Gentle ping @isuruf

**Comment by isuruf** [comment]
Thanks

File: recipe/meta.yaml [hunk 1]
```diff
@@ -449,13 +454,14 @@ outputs:
...
         # Use the same requirements as the top-level requirements
-        - libcxx {{ cxx_compiler_version }}  # [osx]
+        - libcxx-devel {{ cxx_compiler_version }}   # [osx]
         - llvmdev =={{ version }}
         - libxml2
         - zlib
         - zstd
       run:
         - libstdcxx-devel_{{ target_platform }}  # [linux]
+        - libcxx-devel                           # [osx]
```
Comment (isuruf) [inline recipe/meta.yaml:464]: We need to pin the version here. Sorry I missed this.

Comment (h-vetinari) [reply recipe/meta.yaml:464]: Sorry I missed it as well.

**Pull request merged**